x/interchainstaking/keeper/receipt.go

```diff
@@ -25,49 +25,53 @@ const (
 	ICATimeout      = time.Hour * 6
 )
 
-func (k *Keeper) HandleReceiptForTransaction(ctx sdk.Context, txr *sdk.TxResponse, txn *tx.Tx, zone *types.Zone) error {
-	k.Logger(ctx).Info("deposit receipt.", "ischeck", ctx.IsCheckTx(), "isrecheck", ctx.IsReCheckTx())
-	hash := txr.TxHash
+func (k Keeper) HandleReceiptTransaction(ctx sdk.Context, txn *tx.Tx, txHash string, zone types.Zone) error {
```

Comment:
```suggestion
func (k *Keeper) HandleReceiptTransaction(ctx sdk.Context, txn *tx.Tx, txHash string, zone *types.Zone) error {
```
I think we should keep these

Reply:
Resolved in #445